src/openrct2/world/park.c

```diff
@@ -477,6 +477,22 @@ static sint32 park_calculate_guest_generation_probability()
 	return probability;
 }
 
+void get_nearest_peep_spawn(rct2_peep_spawn *spawn, sint32 x, sint32 y)
+{
+	sint32 nearestDist = INT32_MAX;
+	sint32 nearestIndex = MAX_PEEP_SPAWNS;
+	for (sint32 i = 0; i < MAX_PEEP_SPAWNS; i++) {
+		sint32 dx = gPeepSpawns[i].x - x;
+		sint32 dy = gPeepSpawns[i].y - y;
+		sint32 dist = dx * dx + dy * dy;
```

Comment:
check the newest commit, i copied this version from https://github.com/wolfreak99/OpenRCT2/blob/258dddada062c36fb338ba7f6319211eccd411af/src/openrct2/peep/peep.c#L9783 that portion.. however, if you would still prefer the previous i can revert.
also if i do revert or store it in a seperate get_nearest_peep_spawn, would i be able to create a relative get_nearest_park_entrance ?